README.md

```diff
@@ -40,6 +40,10 @@ In the browser `require` and `package.loadlib` try to find a file by making sync
     - `os.execute`
 - [Weak tables](http://www.lua.org/manual/5.3/manual.html#2.5.2)
 
```

Comment:
Could you add a 2nd blank line here? There should be two blank lines at the end of a section. (also need a 2nd empty line after your addition)